src/ontology/mondo-edit.obo

```diff
@@ -465427,8 +465420,8 @@ xref: Orphanet:33110 {source="OMIM:601495"}
 xref: UMLS:C3152144 {source="MEDGEN:463494", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0011096 {source="MONDO:prototype"} ! autosomal agammaglobulinemia
 is_a: MONDO:0015977 {source="MESH:C538056", source="MONDO:Redundant", source="OMIM:601495", source="Orphanet:33110/inferred"} ! agammaglobulinemia
-intersection_of: MONDO:0011096 ! autosomal agammaglobulinemia
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 ! IGHM
+intersection_of: MONDO:0011096 {source="OMIM:601495"} ! autosomal agammaglobulinemia
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5541 {source="OMIM:601495"} ! IGHM
```

Comment:
We usually don't have sources for the "Equivalent To" annotations, so this would be an inconsistency with the rest of the ontology. 
In the big scheme of things, it is not "super bad" I guess.

@matentzn could  please 
1) confirm that it is ok to have a source for this equivalent (and not for all equivalent)
2) confirm that this is how these sources would be displayed in the obo format (note that it looks good in Protege)
Thanks!